datafusion/core/src/physical_plan/display.rs

```diff
@@ -192,11 +209,29 @@ impl<'a, 'b> ExecutionPlanVisitor for IndentVisitor<'a, 'b> {
     }
 }
 
-impl<'a> ToStringifiedPlan for DisplayableExecutionPlan<'a> {
-    fn to_stringified(
-        &self,
-        plan_type: crate::logical_expr::PlanType,
-    ) -> StringifiedPlan {
-        StringifiedPlan::new(plan_type, self.indent().to_string())
+/// Trait for types which could have additional details when formatted in `Verbose` mode
```

Comment:
🤔  maybe eventually we can combine this into `ExecutionPlan` so that everything that implemented `ExecutionPlan` would also need to implement `DisplayAs`. 

That would reduce the duplication across the traits, howe er, it would be an API change and result in non trivial code churn so I think we can consider it as part of a follow on PR

```
trait ExecutionPlan: DisplayAs {
...
}
```

Reply:
Sure but why do you think it won't be trivial? If https://github.com/rust-lang/rust/issues/31844 stabilized it would be much easier of course but as far as I see, the change is tedious but should be straitforward: just split `fmt_as` into as separate `impl`. Do I miss anything here?